helios-benchmarks/build.gradle

```diff
@@ -35,7 +37,7 @@ jmh {
         resultsFile = file(jmhResultsFile)
 }
 
-task updateMasterBenchmark {
+task executeMasterBenchmark {
     doLast {
         jmh.resultsFile = file('src/jmh/resources/master_benchmark.csv')
```

Comment:
It's not a big deal but the `csv` is not an internal resource for the benchmark module. I'd put that file in the root project directory, inside a folder called `benchmarks`

Reply:
solved here https://github.com/47deg/helios/pull/45